x-pack/plugins/security_solution/server/lib/detection_engine/routes/utils.test.ts

```diff
@@ -464,7 +465,7 @@ describe('utils', () => {
         status: 'error',
         lastExecutionDate: foundRule.executionStatus.lastExecutionDate,
         error: {
-          reason: 'read',
+          reason: AlertExecutionStatusErrorReasons.Read,
```

Comment:
Thanks for the cleanup here and above at https://github.com/elastic/kibana/pull/79056/files#diff-f6140d9a8de26e26d80f53651169d01af3446e59c854324b05f492fb708405cbR61 !